### PR TITLE
Change title to HPE Design System

### PR DESCRIPTION
**Pull request by jensy**

### CODE REVIEW

File: aries-site/src/components/seo/Meta.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
 
   return (
     <Head>
-      <title>{title ? `${title} — Aries` : 'Aries | HPE Design System'}</title>
+      <title>{title ? `${title} — HPE Design System` : 'Aries | HPE Design System'}</title>
```
Comment (halocline) [inline aries-site/src/components/seo/Meta.js]: Suggest elimination of Aries: ```<title>{title ? `${title} — HPE Design System` : 'HPE Design System'}</title>```

File: aries-site/src/layouts/content/Head.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import HeadNext from 'next/head';
 
 export const Head = ({ title }) => (
   <HeadNext>
-    <title>{title ? `${title} — Aries` : 'Aries | HPE Design System'}</title>
+    <title>{title ? `${title} — HPE Design System` : 'Aries | HPE Design System'}</title>
```
Comment (halocline) [inline aries-site/src/layouts/content/Head.js]: Suggest elimination of Aries: ```<title>{title ? `${title} — HPE Design System` : 'HPE Design System'}</title>```

**Comment by jensy** [comment]
@halocline Sounds good!

**Comment by halocline** [comment]
> @halocline Sounds good!

@jensy https://github.com/hpe-design/design-system/pull/328 was merged in to correct this issue. Can you look at the production site and confirm the page titles are displaying as expected? If so, I will close this PR. Thank you!

**Comment by jensy** [comment]
@halocline Clicked around and looks good to me! Thank you :)

**Comment by halocline** [comment]
Resolved by https://github.com/hpe-design/design-system/pull/328

**Pull request closed without merging**